method_example/base_class_const.py

```diff
@@ -0,0 +1,21 @@
+class BaseClass:
+    
+    _classvariable = None
+    
+    def __init__(self, val):
+        self.instance_varable = val
+        
+    def show_variable(self):
+        print(f"instance variable update direct from init: {self.instance_varable}")
+        
+    def show_class_variable(cls):
+        print(f"class variable update using class method: {cls._classvariable}")
+        
+    def update_instance_variable(self):
+        self.instance_varable = None
```

Comment:
same comment as above

Reply:
Actually these methods have not called thats why I set the value of None. 
In https://github.com/nawal-neeraj/pytest/blob/feature-methods/method_example/base_class_method.py#L15 this line when I am using method call then I used parameter to update instance variable

Reply:
The point is the method implementation is not making sense. You should review it by yourself and not have such code.

Reply:
Other option is to make it abstract method.